olm-catalog/serverless-operator-index/v4.14/catalog-template.yaml

```diff
@@ -455,7 +458,9 @@ entries:
   - image: registry.redhat.io/openshift-serverless-1/serverless-operator-bundle@sha256:91456907c61952017daf07e376b565472ef86d67a2925f60e189c639ceb68595
     schema: olm.bundle
   - schema: olm.bundle
-    image: quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-136/serverless-bundle@sha256:7752af022214e20d134fdb87bf09b9d3c0de1c7ecba8310a95ea32881a02afe6
+    image: registry.ci.openshift.org/knative/serverless-bundle@sha256:07387fa6c4294c9933aa94865b07a647aadc856b0d7f9cf8e9b36ba901da44e8
+  - schema: olm.bundle
+    image: quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-136/serverless-bundle@sha256:c14c5f0be48b343bcc9006356334ddd0780b652a3639d893390f102d7e962e12
```

Comment:
registry.ci.openshift.org/knative/serverless-bundle@sha256:07387fa6c4294c9933aa94865b07a647aadc856b0d7f9cf8e9b36ba901da44e8 and quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-136/serverless-bundle@sha256:c14c5f0be48b343bcc9006356334ddd0780b652a3639d893390f102d7e962e12 are both 1.36 bundles. So I think we need on successful 1.37 bundle & fbc build to have it here updated correctly

Reply:
Addressed in 28b777d07bdb1d7db778253e6b6c4c798db25c3d